app/utils/member_list.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import MemberListRow from 'app/components/custom_list/member_list_row';
+import {displayUsername} from 'service/utils/user_utils';
+
+export const loadingText = {
+    id: 'mobile.loading_members',
+    defaultMessage: 'Loading Members...'
+};
+
+export function createMembersSections(data) {
+    const sections = {};
+    data.forEach((d) => {
+        const name = d.username;
+        const sectionKey = name.substring(0, 1).toUpperCase();
+
+        if (!sections[sectionKey]) {
+            sections[sectionKey] = [];
+        }
+
+        sections[sectionKey].push(d);
+    });
+
+    return sections;
+}
+
+export function renderMemberRow(user, sectionId, rowId, preferences, theme, selectable, onPress, onSelect) {
```

Comment:
I think this belongs in the scenes that use the```MemberListRow``` so that the scene can control the rendering. I know that in the manage members scene we need to disable actions for the current user in the list so we would end up duplicating this code to the scene. Just seems out of place to me, thoughts?